frontend/src/components/Navbar.tsx

```diff
@@ -0,0 +1,186 @@
+import {
+  Box,
+  Flex,
+  Center,
+  IconButton,
+  Button,
+  Link,
+  Image,
+  Stack,
+  Collapse,
+  useColorModeValue,
+  useBreakpointValue,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import colors from "../styles/colors";
+import Logo from "../assets/logo.png";
+import { HamburgerIcon, CloseIcon } from "@chakra-ui/icons";
+import { Link as ReactRouterLink } from "react-router-dom";
+
+export default function Navbaor() {
```

Comment:
Component name typo. You can add spell check into VSCode: I use [Code Spell Checker by streetsidesoftware](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker). It should handle camel case words like `twoWords` so that it doesn't trigger the typo line.